test/generate.test.js

```diff
@@ -11,16 +11,17 @@ var test = require('tap').test,
 var update = process.env.UPDATE;
 var emptyPNG = new mapnik.Image(1, 1).encodeSync('png');
 
+const fixtures = glob.sync(path.resolve(path.join(__dirname, '/fixture/svg/*.svg'))).map(function(im) {
+    return {
+        svg: fs.readFileSync(im),
+        id: path.basename(im).replace('.svg', '')
+    };
+});
+
```

Comment:
Profiling locally showed that the `bench` tests below were primarily blocked on `fs.readFileSync`, since it was - before this change - called for each task put into the queue. This changes things so the fixtures are only read once and allows the benchmarks to benchmark spritezero itself rather than `fs.readFileSync`. In local testing I see:

## before

> generateLayout bench (concurrency=4,x20) # time=1512.339ms

## after

>  generateLayout bench (concurrency=4,x20) # time=1082.892ms